guides/maintainer/github-admin-processes.md

```diff
@@ -0,0 +1,43 @@
+# GitHub Administration Process
+
+This document outlines how the GitHub administration process is handled within the OpenTelemetry organization.
+
+## Principles
+
+1. **Transparency**: Document and communicate all changes
+2. **Collaboration**: Involve affected SIGs in decision-making processes
+3. **Responsiveness**: Move quickly on maintainer requests
+
+## Guidelines
+
+When possible, changes should be implemented by sending a PR to [open-telemetry/admin](https://github.com/open-telemetry/admin) (updating the appropriate Terraform files).
```

Comment:
Maybe an obvious comment but we could say that we will restrict permissions to manually change settings whenever possible to have a single source of truth on open-telemetry/admin?

Reply:
We should call out here, that the repo is private but accessible by all maintainers, so the "transparency" is project-internal and not public

Reply:
> Maybe an obvious comment but we could say that we will restrict permissions to manually change settings whenever possible to have a single source of truth on open-telemetry/admin?

I've sent a series of PRs to remove admin permission from the maintainers.

* https://github.com/open-telemetry/admin/pull/115
* https://github.com/open-telemetry/admin/pull/116
* https://github.com/open-telemetry/admin/pull/117
* https://github.com/open-telemetry/admin/pull/118
* https://github.com/open-telemetry/admin/pull/119
* https://github.com/open-telemetry/admin/pull/120

Reply:
> Maybe an obvious comment but we could say that we will restrict permissions to manually change settings whenever possible to have a single source of truth on open-telemetry/admin?

It's a bit more than this even, if you update something manually, the next time terraform runs your change will get wiped out.

Reply:
> We should call out here, that the repo is private but accessible by all maintainers

good point, added this